dashboards/relaysvc/relaysvc.json

```diff
@@ -0,0 +1,1217 @@
+          "legendFormat": "{{reason}}",
+          "range": true,
+          "refId": "A"
+        }
+      ],
+      "title": "Connection Request Rejected",
+      "type": "timeseries"
+    },
+    {
+      "datasource": {
+        "type": "prometheus",
+        "uid": "${DS_PROMETHEUS}"
+      },
+      "description": "",
+      "fieldConfig": {
+        "defaults": {
+          "color": {
+            "mode": "palette-classic"
+          },
+          "custom": {
+            "axisCenteredZero": false,
+            "axisColorMode": "text",
+            "axisLabel": "",
+            "axisPlacement": "auto",
+            "barAlignment": 0,
+            "drawStyle": "line",
+            "fillOpacity": 0,
+            "gradientMode": "none",
+            "hideFrom": {
+              "legend": false,
+              "tooltip": false,
+              "viz": false
+            },
+            "lineInterpolation": "linear",
+            "lineWidth": 1,
+            "pointSize": 5,
+            "scaleDistribution": {
+              "type": "linear"
+            },
+            "showPoints": "auto",
+            "spanNulls": false,
+            "stacking": {
+              "group": "A",
+              "mode": "none"
+            },
+            "thresholdsStyle": {
+              "mode": "off"
+            }
+          },
+          "mappings": [],
+          "thresholds": {
+            "mode": "absolute",
+            "steps": [
+              {
+                "color": "green"
+              },
+              {
+                "color": "red",
+                "value": 80
+              }
+            ]
+          }
+        },
+        "overrides": []
+      },
+      "gridPos": {
+        "h": 8,
+        "w": 12,
+        "x": 12,
+        "y": 42
+      },
+      "id": 18,
+      "options": {
+        "legend": {
+          "calcs": [],
+          "displayMode": "list",
+          "placement": "bottom",
+          "showLegend": true
+        },
+        "tooltip": {
+          "mode": "single",
+          "sort": "none"
+        }
+      },
+      "targets": [
+        {
+          "datasource": {
+            "type": "prometheus",
+            "uid": "${DS_PROMETHEUS}"
+          },
+          "editorMode": "code",
+          "expr": "histogram_quantile(0.5, sum(rate(libp2p_relaysvc_connection_duration_seconds_bucket[$__rate_interval])) by (le))",
+          "legendFormat": "50th percentile",
+          "range": true,
+          "refId": "A"
+        },
+        {
+          "datasource": {
+            "type": "prometheus",
+            "uid": "${DS_PROMETHEUS}"
+          },
+          "editorMode": "code",
+          "expr": "histogram_quantile(0.99, sum(rate(libp2p_relaysvc_connection_duration_seconds_bucket[$__rate_interval])) by (le))",
+          "hide": false,
+          "legendFormat": "99th percentile",
+          "range": true,
+          "refId": "C"
+        }
+      ],
+      "title": "Connection Duration",
```

Comment:
This diagram seems to be missing the unit.

I'm also not sure how useful it is, the graph looks kind of wild on my instance:
<img width="858" alt="image" src="https://user-images.githubusercontent.com/1478487/223333912-0fbdf7ed-03bd-4954-9923-837349347670.png">


Maybe it's easier if we just track the average:
```
rate(libp2p_relaysvc_connection_duration_seconds_sum[$__range])/rate(libp2p_relaysvc_connection_duration_seconds_count[$__range])
```
(Please double-check if `$__range` is the appropriate variable here)

<img width="837" alt="image" src="https://user-images.githubusercontent.com/1478487/223333809-656c88b1-18e4-41fe-8616-d905d6b8141e.png">

Reply:
I'm keeping it range because I think it is most informative, while rate_interval will give you average in that time period the graph is very spiky and it is also an average over all connections in that period so the spike doesn't necessarily mean it's something is wrong. 
On the other hand increasing the dashboard range will change this graph which seems wrong. 
<img alt="Screenshot 2023-03-07 at 12 32 24 PM" src="https://user-images.githubusercontent.com/2179993/223350137-0880923c-06f9-4d24-83c0-62bcd65b15d9.png">

Reply:
changed the label to rolling average.